powershell/Docs/ReleaseNotes.md

```diff
@@ -1,14 +1,18 @@
 # Release Notes
 
+## 0.6.1
```

Comment:
1) Documented the new change.
2) Looks like I forgot to document the Data property change for 0.6.0. I added that.
3) Took a quick pass through and made the release statements consistent in format. Added some backticks around the commands/switches/parameters and changed the sentence structure to make it easier to read.